bundles/org.openhab.binding.echonetlite/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,366 @@
+				<option value="Used (horizontal)">Used (horizontal)</option>
+				<option value="Used (vertical and horizontal)">Used (vertical and horizontal)</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="airFlowDirectionVertical">
+		<item-type>String</item-type>
+		<label>Air Flow Direction Vertical</label>
+		<description>Air Flow Direction Vertical</description>
+		<state>
+			<options>
+				<option value="Uppermost">Uppermost</option>
+				<option value="Lowermost">Lowermost</option>
+				<option value="Mid-uppermost">Mid-uppermost</option>
+				<option value="Mid-lowermost">Mid-lowermost</option>
+				<option value="Central">Central</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="airFlowDirectionHorizontal">
+		<item-type>String</item-type>
+		<label>Air Flow Direction Horizontal</label>
+		<description>Air Flow Direction Horizontal</description>
+		<state>
+			<options>
+				<option value="XXXOO">XXXOO</option>
+				<option value="OOXXX">OOXXX</option>
+				<option value="XOOOX">XOOOX</option>
+				<option value="OOXOO">OOXOO</option>
+				<option value="XXXXO">XXXXO</option>
+				<option value="XXXOX">XXXOX</option>
+				<option value="XXOXX">XXOXX</option>
+				<option value="XXOXO">XXOXO</option>
+				<option value="XXOOX">XXOOX</option>
+				<option value="XXOOO">XXOOO</option>
+				<option value="XOXXX">XOXXX</option>
+				<option value="XOXXO">XOXXO</option>
+				<option value="XOXOX">XOXOX</option>
+				<option value="XOXOO">XOXOO</option>
+				<option value="XOOXX">XOOXX</option>
+				<option value="XOOXO">XOOXO</option>
+				<option value="XOOOO">XOOOO</option>
+				<option value="OXXXX">OXXXX</option>
+				<option value="OXXXO">OXXXO</option>
+				<option value="OXXOX">OXXOX</option>
+				<option value="OXXOO">OXXOO</option>
+				<option value="OXOXX">OXOXX</option>
+				<option value="OXOXO">OXOXO</option>
+				<option value="OXOOX">OXOOX</option>
+				<option value="OXOOO">OXOOO</option>
+				<option value="OOXXO">OOXXO</option>
+				<option value="OOXOX">OOXOX</option>
+				<option value="OOOXX">OOOXX</option>
+				<option value="OOOXO">OOOXO</option>
+				<option value="OOOOX">OOOOX</option>
+				<option value="OOOOO">OOOOO</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="operationMode">
+		<item-type>String</item-type>
+		<label>Operation Mode</label>
+		<description>Operation Mode</description>
+		<state>
+			<options>
+				<option value="Automatic">Automatic</option>
+				<option value="Cooling">Cooling</option>
+				<option value="Heating">Heating</option>
+				<option value="Dry">Dry</option>
+				<option value="Fan">Fan</option>
+				<option value="Other">Other</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="setTemperature">
+		<item-type>Number</item-type>
+		<label>Set Temperature</label>
+		<description>Set Temperature</description>
+		<category>Heating</category>
```

Comment:
If this is a temperature setpoint, then category **Temperature** seems more appropriate:
```suggestion
		<category>Temperature</category>
```